package.xml

```diff
@@ -1,18 +1,24 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>teleop_twist_keyboard</name>
   <version>0.6.0</version>
-  <description>Generic keyboard teleop for twist robots.</description>
+  <license>BSD</license>
+  <description>
+    A robot-agnostic teleoperation node to convert keyboard commands to Twist
+    messages.
+  </description>
 
   <maintainer email="namniart@gmail.com">Austin Hendrix</maintainer>
-
-  <license>BSD</license>
+  <author>Graylin Trevor Jay</author>
 
   <url type="website">http://wiki.ros.org/teleop_twist_keyboard</url>
+  <url type="development">https://github.com/ros2/teleop_twist_keyboard</url>
+  <url type="bugtracker">https://github.com/ros2/teleop_twist_keyboard/issues</url>
```

Comment:
If we add urls to the repo we should remove the "website" one because they refer to different pieces of code. Usually we do neither So I'd suggest not adding these 2 url tags